src/users/users.service.ts

```diff
@@ -15,4 +16,30 @@ export class UsersService {
       photo: '',
     };
   }
+
+  getPhotoUrl(user: User) {
+    if (user.photo) {
+      return user.photo;
+    } else {
+      // TODO: Create new photo, see old code
+      return '';
+    }
+  }
+
+  toUserDto(user: User): UserInfoDto {
+    if (user === undefined) {
+      this.logger.warn('toUserDto recieved undefined argument!');
+      return null;
+    }
+    if (user === null) {
+      this.logger.warn('toUserDto recieved null argument!');
+      return null;
+    }
```

Comment:
I think this could be moved into one as we don't really need to distinguish between `undefined` and `null`

Reply:
done